libs/FileManagerKit/tests/FileManagerKit_test.cpp

```diff
@@ -81,3 +81,30 @@ TEST_F(FileSystemTest, removeNotExistingDirectory)
 	auto removed = FileManagerKit::remove(path_B);
 	ASSERT_FALSE(removed);
 }
+
+TEST_F(FileSystemTest, clear)
+{
+	std::ofstream(path_A).put('a');
+
+	auto initial_file_size	= std::filesystem::file_size(path_A);
+	auto initial_free_space = std::filesystem::space(path_A).free;
+
+	EXPECT_NE(initial_file_size, 0);
+
+	FileManagerKit::clear(path_A);
+
+	auto final_file_size  = std::filesystem::file_size(path_A);
+	auto final_free_space = std::filesystem::space(path_A).free;
```

Comment:
I'm not sure I understand why you want to test it like this.

why not just write to the file, measure the size, clear it, measure again?

Reply:
It's to verify that it is not just a truncation of a file: move the EOF character to the beginning of file and kee the previous data allocated but inaccessible.